packages/language-server/package.json

```
@@ -61,7 +61,7 @@
   },
   "peerDependencies": {
     "prettier": "^3.0.0",
-    "prettier-plugin-astro": "^0.11.0"
+    "prettier-plugin-astro": "^0.11.0 || ^0.12.0"
```

Since it's unlikely that the API will change, I think we could even use `>=0.11.0`

---

If you're confident about it, I'm also fine too 😄 
```suggestion
    "prettier-plugin-astro": ">=0.11.0"
```